multigeiger/chkhardware.cpp

```diff
@@ -0,0 +1,19 @@
+// Check if we have LoRa hardware
+
+// Test pin 26 (that is LORA_DIO0) to be low.
+#include <Arduino.h>
+
+#define HWTESTPIN 26
+
+static bool haveLoRa = false;
+
+// Test pin is LoRa_DIO0. If LoRa chip is present, this pin is low, otherwise
+// it is an open pin with pullup
+bool init_hwtest(void) {
+  pinMode(HWTESTPIN, INPUT_PULLUP);       
+  delay(200);
+  if(!digitalRead(HWTESTPIN)) {       // low => LoRa chip detected
+    haveLoRa = true;
+  }  
+  return haveLoRa;
```

Comment:
```
return !digitalRead(HWTESTPIN);
```

also: either check this only once or kill that static variable, guess both together doesn't make sense.

Reply:
but a `int board` variable would be even better.

Reply:
I prefer a bool for easier tests and if/else and we only have 2 boards ( if there are sometimes more boards, we have to change the routine anyway). 
Let's call it 'bool loraBoard'. I want to have the 'lora' there , because we test the presence of the LoRa chip.

Reply:
I still think an enum is the more appropriate approach than a bool.

This is not about True/False, but about having a specific board or not (which could, even if we only have 2 now, be > 2 at any time soon [and was > 2 until recently also]).